javascript/javascript1/week3/homework4.js

```diff
@@ -0,0 +1,74 @@
+// Smart-ease - Back to the basics!
+// NOnoN0nOYes (Note taking app)
+
+// The saveNote function should push an object to the notes array with the keys content and id.
+
+const notes = [];
+/**
+ *
+ *
+ * @param {string} content
+ * @param {number} id
+ */
+
+function saveNote(content, id) {
+  notes.push({
+    content: content,
+    id: id,
+  });
+}
+
+saveNote("Pick up groceries", 1);
+saveNote("Do laundry", 2);
+
+//unique feature of colored log
+console.log("\x1b[31m%s\x1b[0m", "\n\t SaveNote task");
+console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
+
+// Get a note
+
+function getNote(id) {
+  for (let i = 0; i < notes.length; i++) {
+    if (notes[i].id === id) {
+      return notes[i];
+    }
+  }
+}
```

Comment:
Here is a good place to return an error message, if the return inside the loop doesn't give a result.
Eg. "Sorry, we couldn't find a note with the provided id".

Reply:
done!